recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "python-dateutil" %}
-{% set version = "2.9.0" %}
+{% set version = "2.9.0.post0" %}
```

Comment:
Post releases, if done correctly, do not bring new code but fix packaging issues, like wheels. We should always skip those unless upstream abused PEP440.  For more info please see https://github.com/conda-forge/python-dateutil-feedstock/pull/25